monkestation/strings/random_crimes.txt

```diff
@@ -59,3 +59,16 @@ Reached Skate Nirvana, Flubbed Up
 Too Cool
 Cooking Reality-Bending Food
 Recycling
+Caused a Intergalactic War Between NanoTrasen And The Clown Union, Due To Adding Silentium To a Banana
+Stun Prodded a Clown And Spacing Said Clown Out a Airlock
+Blew Up a Station With a Supermatter Delamination
+Murderboned TOO HARD
+Drugged a Station's Crew With Floorpills
+Has a Engine Addiction that Causes More Harm Then Good
+Unleashed a Chaos Dunk On 15 Different Space Stations, They Were Stopped by a Security Officer On One Station Before They Did Another Chaos Dunk On Said Station, Many Know Them As "The Chaos Dunk Maniac"
+Was Caught Killing Other Crimers And Was Arrested For Vigilante Justice
+Was a Intern That Stole Very Confidential [REDACTED] NT Documents From Central Command, Was Swiftly Fired By Nanotrasen and Arrested By Nanotrasen Central Command Security
+Stole a Ancient Clown Banana Artifact From The Clown Union
+Bio-Terrorism
+Released a Mutegen Virus That Turned Many into Mutant Clowns And Flooded The Station With Plasma
```

Comment:
Spelling error on this one

Reply:
fixed the mutagen spelling error
and a another spelling error